### DISCUSSION TITLE
Don't notify user of completed download when opening folder

### DISCUSSION THREAD

**Discussion by mskelton** [zoidsh/meru#259] [Feature Requests] [answered]
When the `Open Folder When Done` setting is enabled, it is probably not necessary to send the push notification since the file explorer/finder will already be opened resulting in two "notification" type things.

**Comment by timche** [comment]
Good catch, will implement that. Thanks!

**Reply by mskelton** [reply]
If you don't mind I'd like to take this on as a way of getting back into the project!

**Reply by timche** [reply]
Sure thing. Feel free 😄

**Comment by mskelton** [comment, marked as answer]
Fixed via #260
